### PR TITLE
murmur on debian

### PR DESCRIPTION
**Pull request by markzz**
Resubmission from closed pull request #239.

### CODE REVIEW

**Comment by thetechstewart** [comment]
@markzz - Thanks for resubmitting, and my apologies for overlooking your previous PR.  I'm reviewing my github alert settings to make sure that doesn't happen again.

Regarding this guide, I've run through a tech test, and it seems that the server set up portion is solid.  However, it does not appear that there are any instructions for setting up the clients to connect to the server.  Could you append a section for setting up the client connections to the guide?  Once that has been added, I think this would be a solid addition to the Linode Guides and Tutorials site.

**Comment by markzz** [comment]
I apologize for how long this took me, but I added a client connection portion now.

File: docs/applications/voip/setup-a-mumble-server-on-debian.md [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+contributor:
+    name: Your Name
+    link: Github/Twitter Link
+---
+
+Mumble is an open-source VoIP client, designed for gamers, that requires a server for all clients to connect to. This guide instructs you on how to install and configure the Mumble server (also called Murmur) on Debian.
+
+No changes need to be made to the system beforehand if you follow this guide. 
+
+Througout this guide, all commands are run as root.  If you are not root user, you can prepend each command with "sudo."
```
Comment (alexfornuto) [inline docs/applications/voip/setup-a-mumble-server-on-debian.md]: Please replace this with the blurb from our style guide:

```
{: .note}
>
>The steps required in this guide require root privileges. Be sure to run the steps below as **root** or with the `sudo` prefix. For more information on privileges see our [Users and Groups](/docs/tools-reference/linux-users-and-groups) guide.
```

File: docs/applications/voip/setup-a-mumble-server-on-debian.md [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+
+No changes need to be made to the system beforehand if you follow this guide. 
+
+Througout this guide, all commands are run as root.  If you are not root user, you can prepend each command with "sudo." 
+
+## Server Side
+
+### Install and Simple Setup
+
+1. Since Murmur is in the official Debian repositories, you can just use apt-get to install it.  Be careful though, the package is `mumble-server` and not `murmur`. Enter the following command (remember to prepend with sudo if needed):
```
Comment (alexfornuto) [inline docs/applications/voip/setup-a-mumble-server-on-debian.md]: I would kill off that last sentence. It's obvious that you need to run the command to do the needful.

File: docs/applications/voip/setup-a-mumble-server-on-debian.md [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+
+	apt-get install mumble-server
+
+2. After installation, you can use `dpkg-reconfigure` to configure the initial setup.
+
+The first question asked during system config will be whether or not you want the server to run at boot.  This is identical to using the command `systemctl enable mumble-server`.
```
Comment (alexfornuto) [inline docs/applications/voip/setup-a-mumble-server-on-debian.md]: This is true for Debian 8 but not Debian <8. The other commands below are for both versions, so I assume you want this one to be correct for older versions as well.

**Comment by markzz** [comment]
Any updates?

**Comment by markzz** [comment]
Any updates?

**Comment by cwlinode** [comment]
@markzz I'm currently finishing up with this. I'll have the final draft ready shortly. Then we'll publish and send your payment.

**Comment by cwlinode** [comment]
@markzz Could you please merge the changes from your branch markzz-murmur-debian in your master.

**Comment by markzz** [comment]
Okay, I believe it's done.

**Comment by markzz** [comment]
Now that this is published, when should I expect word about payment?

**Comment by cwlinode** [comment]
@markzz Please confirm your Paypal address with contribute@linode.com.

**Pull request merged**